.travis.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         -
             stage: Static Code Analysis
             php: 7.2
-            env: COMPOSER_FLAGS="--no-dev --prefer-stable"
```

Comment:
it was `--no-dev` to not run `composer-require-checker` with dev dependencies of main composer file installed

Reply:
I need to install dev dependencies so that PHPStan does not complain about missing classes. I think Composer Require Checker ignores dev dependencies anyway.